app/views/shared/_users.html.erb

```diff
@@ -0,0 +1,10 @@
+<div id="each-user">
+    <div class="boxes img-card">
+      <img src=<%= user.photo %> alt="profile" class="img-card">
+    </div>
+    <div class=" boxes user-card" >
+      <a href="/users/<%= user.id%>"> <h2 class="user-name"> <%= user.name %></h2></a>
+      <p class="post-counter">Number of posts: <%= user.update_user_posts_counter%> </p>
+    </div>
+ 
+</div>
```

Comment:
- [ ] **[OPTIONAL]** Please consider taking a look at your Posts counter! Currently, when I create a new post, it doesn't get updated. 🤓 🔧 

![image](https://github.com/joyapisi/blog_app/assets/77069011/bf761b84-b7a6-47b9-97dc-1894301d2054)